cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -100,30 +102,35 @@ Hadoop components, as required:
+|                | |cdh-title|                     | |cdh-versions| |note-4|               |
 +                +---------------------------------+---------------------------------------+
-|                | Hortonworks Data Platform (HDP) | |hdp-versions|                        |
+|                | Hortonworks Data Platform (HDP) | |hdp-versions| |note-4|               |
 +                +---------------------------------+---------------------------------------+
 |                | MapR                            | |mapr-versions|                       |
 +                +---------------------------------+---------------------------------------+
 |                | Amazon Hadoop (EMR)             | |emr-versions|                        |
 +----------------+---------------------------------+---------------------------------------+
 | **Spark**      | Apache                          | Versions 1.2.x through 1.6.x          |
 +                +---------------------------------+---------------------------------------+
-|                | |cdh-title|                     | |cdh-versions|                        |
+|                | |cdh-title|                     | |cdh-versions| |note-4|               |
 +                +---------------------------------+---------------------------------------+
-|                | Hortonworks Data Platform (HDP) | |hdp-versions|                        |
+|                | Hortonworks Data Platform (HDP) | |hdp-versions| |note-4|               |
 +                +---------------------------------+---------------------------------------+
 |                | MapR                            | |mapr-versions|                       |
 +                +---------------------------------+---------------------------------------+
 |                | Amazon Hadoop (EMR)             | |emr-versions|                        |
 +----------------+---------------------------------+---------------------------------------+
 
-**Note:** Component versions shown in these tables are those that we have tested and are
+**Note 1:** Component versions shown in these tables are those that we have tested and are
 confident of their suitability and compatibility. Later versions of components may work,
 but have not necessarily been either tested or confirmed compatible.
 
-**Note:** Certain CDAP components need to reference your *Hadoop*, *YARN*, *HBase*, and
+**Note 2:** Certain CDAP components need to reference your *Hadoop*, *YARN*, *HBase*, and
 *Hive* cluster configurations by adding those configurations to their class paths.
 
-**Note:** *Hive 0.12* is not supported for :ref:`secure cluster configurations <admin-security>`.
+**Note 3:** *Hive 0.12* is not supported for :ref:`secure cluster configurations <admin-security>`.
+
+.. _admin-manual-hadoop-compatibility-matrix-note-4:
+
+**Note 4:** An upcoming release of CDAP (scheduled for CDAP 4.3) will drop support for all
+versions older than *CDH 5.4.11* or *HDP 2.5.0.0*.
```

Comment:
👍 Ideally it would be nice if there was a link to an announcement from the distros dropping support.